### PR TITLE
Solutions to leetcode57

### PR DESCRIPTION
**Pull request by janicelmm**
Solutions to leetcode57

### CODE REVIEW

File: src/Leetcode57InsertInterval.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.List;
+import java.util.PriorityQueue;
+import java.util.Queue;
+
+public class Leetcode57InsertInterval {
+	public static int[][] insert(int[][] intervals, int[] newInterval) {
+		Comparator<Integer[]> com = (a, b) -> a[0].compareTo(b[0]);
+		Queue<Integer[]> sort = new PriorityQueue<Integer[]>(com);
+
+        for (int i = 0; i < intervals.length; i++)
```
Comment (Ewlai) [inline src/Leetcode57InsertInterval.java:12]: fix indentation

**Review by Ewlai — CHANGES_REQUESTED** [review]
Please see my notes

File: src/Leetcode57InsertInterval.java (hunk 1 above)
Comment (janicelmm) [reply src/Leetcode57InsertInterval.java:12]: Resolved

**Review by Ewlai — APPROVED** [review]
LGTM

**Pull request merged**